samples/basic/blinky_pwm/prj.conf

```diff
@@ -2,4 +2,6 @@ CONFIG_STDOUT_CONSOLE=y
 CONFIG_PRINTK=y
 CONFIG_PWM=y
 CONFIG_LOG=y
+CONFIG_LOG_PRINTK=y
+CONFIG_LOG_IMMEDIATE=y
```

Comment:
Why? At that point you might as well disable `CONFIG_LOG`?

Reply:
The PWM driver prints logs which are useful for understanding what the hardware is doing, and are a hint to the user to grep for the location of the driver for their hardware.

Reply:
(But I am using IMMEDIATE so printk calls are properly serialized with logging output and do not interleave on the console)